babel/src/board.js

```diff
@@ -284,6 +284,18 @@ export class Board {
         return hasWideStyleMatch || _.some(_.map(chunks), hasMatchInPairOfRows);
     }
 
+    swap(swapOrbs) {
+        let [[x1, y1], [x2, y2]] = swapOrbs;
+        let orbsBefore = _.cloneDeep(this.orbs);
```

Comment:
This `_.cloneDeep()` brought up a new lesson for me. Confirmed [here](http://stackoverflow.com/questions/2612802/how-to-clone-or-copy-a-list-in-python), I realized that just copying something with an assignment and then changing that copy will change both the copy and the original.